### ISSUE TITLE
Lack of precision in the readme concerning weights format

### ISSUE THREAD

**Issue by Alderi-Tokori** [gcp/leela-zero#22]
In the readme, under the "Weights format" section, it is not specified in which order the value head and the policy head should be put into the weight file.

Furthermore, it is not said in which "order" the weights should be put in each weight line: by browsing the next branch, I saw in tfprocess.py the following comment:
```
#TF
# [filter_height, filter_width, in_channels, out_channels]
# Leela
# (output, input, filter_size, filter_size]
```

Which indicates that leela reads weights in a different order than how tensorflow formats them. These details should be specified in the readme so people can properly format the weights.txt file.

**Comment by gcp** [comment]
I will update these when "next" is fully functional. The order (weights and layers) corresponds to the Caffe protobuff files that are in the repo.

**Comment by Alderi-Tokori** [comment]
To be sure, does the current master branch reads weight in the same order, namely [output, input, filter_size, filter_size] ? I'm using tensorflow so I want to be sure I'm exporting the weights properly.

**Comment by gcp** [comment]
Yes: [output, input, filter_size, filter_size] 

I haven't tested the export in "next" yet, I've never written TensorFlow before and BatchNorm is being a bit tricky to get right.

**Comment by Alderi-Tokori** [comment]
I'm having the same issue concerning batch_norm, I used the code provided at https://stackoverflow.com/a/34634291 and I seem to be able to retrieve the learned means and variances through tf.get_collection(tf.GraphKeys.GLOBAL_VARIABLES), though I haven't properly tested it yet.

**Comment by gcp** [comment]
I was able to use the built-in layer (which can forward to cuDNN if conditions are correct, rather important) with some use of variable_scope and then picking out the names. Code is in the "next" branch. I think the export is now complete so after letting it running for a bit it should become obvious soon whether I got the format description and weight order right or not!

**Comment by gcp** [comment]
The order for the innerproduct/fully connected layers must also be reversed. I documented this in the readme in https://github.com/gcp/leela-zero/commit/9d91bdd3fd84ad22eb49129d41c89861ea1f1e98.

The "next" branch now works and I merged the TensorFlow learning code to master.
